packages/elements/src/chart/elements/chart.ts

```diff
@@ -325,6 +324,17 @@ export class Chart extends BasicElement {
     return isNaN(result) ? undefined : result;
   }
 
+  /**
+   * Callback beforeUpdate event from Chartjs
+   * @param chart Chart.js instance
+   * @returns {void}
+   */
+  protected beforeUpdate = (chart: ChartJS): void => {
+    // set global config again to prevent unloaded font in time.
```

Comment:
From our discussion, I think this `setGlobalConfig` is used in case the font doesn't load when the chart updating, it doesn't mean this approach is used to prevent unloaded font.

Reply:
@Theeraphat-Sorasetsakul Have you fixed this?

Reply:
Sorry for late update. I fixed. please take a look.